Client.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -111,10 +111,10 @@
     {
       "identity" : "rust-components-swift",
       "kind" : "remoteSourceControl",
-      "location" : "https://github.com/mozilla/rust-components-swift.git",
+      "location" : "https://github.com/ecosia/rust-components-swift/",
       "state" : {
-        "revision" : "d80080cd008b36b2da81b3242a8ffb37d7499adc",
-        "version" : "122.0.2"
+        "branch" : "122.0.2_Glean_dependency_removed",
```

Comment:
Is this approach of using a separate branch new? Do we plan on keeping it instead of updating main of the ecosia fork?

Reply:
Nothing new, we are doing the same with the current Glean Removal approach we did in prod.
We have forked the package and perform the changes there